import/vufind.properties

```diff
@@ -6,3 +6,8 @@
 # look in the VuFind 2.x-style directory path, but you can uncomment the line
 # below for compatibility with VuFind 1.x.
 #vufind.config.relative_path = web/conf
+
+# coordinate error log location - where to put coordinate indexing error logs
+# if not set, the default path will be VUFIND_LOCAL_DIR/import, and failing that,
+# VUFIND_HOME/import.
+coordinate.log.path = /usr/local/vufind/import
```

Comment:
Let's comment this out for now to use the default behavior by default.

Reply:
Ok. Will update this.